src/App.tsx

```diff
@@ -27,10 +26,24 @@ class App extends Component<{}, State> {
     }
   }
 
+  toggleDarkMode = () => {
```

Comment:
I love this function so much, and how it's being used to alter a classname. Wild.